src/Google.Cloud.Functions.ConformanceTests/create-standalone.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+# This script is designed to be used in the buildpack validation
+# hook. Its purpose is to provide a function which can be built
+# using a buildpack, but with the latest (potentially unreleased)
+# version of the Functions Framework.
+
+# Steps:
+#
+# - Clear out tmp
```

Comment:
[nit] You could skip clearing temp and moving the function source code around if you like. You'll get a throwaway GitHub runner machine per job, so each test run will get its own copy of the repo and clean state.

Reply:
True, but it makes it harder to test - given that there's very little to copy, I'll keep it for now.